source/renderer/app/i18n/locales/ja-JP.json

```diff
@@ -512,6 +518,7 @@
   "static.about.copyright": "Input Output HK Limited. Licensed under",
   "static.about.license": "Apache 2.0 license",
   "static.about.title": "Daedalus",
+  "static.dummy.markdown": "!!!Content",
```

Comment:
Can you add JP translations?

Reply:
@DeeJayElly This is out of scope of this PR.